Framework/API/inc/MantidAPI/BoxController.h

```diff
@@ -321,20 +322,14 @@ class DLLExport BoxController {
 
   /** Return the total number of MD Boxes, irrespective of depth */
   size_t getTotalNumMDBoxes() const {
-    size_t total = 0;
-    for (size_t depth = 0; depth < m_numMDBoxes.size(); depth++) {
-      total += m_numMDBoxes[depth];
-    }
-    return total;
+    return std::accumulate(m_numMDBoxes.cbegin(), m_numMDBoxes.cend(),
+                           size_t{0}, std::plus<size_t>());
```

Comment:
The overload of `std::accumulate` without `BinaryOperation` should work here as well since it's summation, but it's probably good to state explicitly what the accumulation does.